habitat/datasets/rearrange/sim_utilities.py

```diff
@@ -0,0 +1,838 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os.path as osp
+from typing import Any, Dict, List, Optional, Union
+
+import magnum as mn
+import numpy as np
+
+import habitat_sim
+
+
+class DebugVisualizer:
+    """
+    Support class for simple visual debugging of a utility simulator.
+    Assumes the default agent (0) is a camera (i.e. there exists an RGB sensor coincident with agent 0 transformation).
+    """
+
+    def __init__(
+        self,
+        sim: habitat_sim.Simulator,
+        output_path: str = "visual_debug_output/",
+        default_sensor_uuid: str = "rgb",
+    ) -> None:
+        """
+        Initialize the debugger provided a Simulator and the uuid of the debug sensor.
+        NOTE: Expects the debug sensor attached to and coincident with agent 0's frame.
```

Comment:
How about assert this? Or enforce this assumption directly by setting the agent and sensor scene nodes directly?